FirebaseDatabase/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # v7.5.0
 - [added] Implmement `queryStartingAfterValue` and `queryEndingBeforeValue` for FirebaseDatabase query pagination.
 - [added] Added `DatabaseQuery#getData` which returns data from the server when cache is stale (#7110).
+- [added] Check for active-listener in-memory cache values in `FIRDatabaseQuery#getDataWithCompletionBlock` (#7312).
```

Comment:
Please use something similar to what you have on Android. It should describe the user-visible behavior change in terms the user understands.

I think you also need to move this to the next release. This unfortunately is now v7.15.1.